components/style/values/computed/mod.rs

```diff
@@ -28,6 +28,9 @@ pub struct Context<'a> {
     pub style: ComputedValues,
 }
 
+// empty type; helps keep the code for to_computed_value and to_specified_value the same
+pub struct UncomputeContext;
```

Comment:
Since you didn't need any context information for the `to_specified_value` implementations you've added, it's not clear whether we'll have any kind of useful context to pass in at the point we'll be calling `to_specified_value` from Gecko, and we no longer need it for the mako templates, let's remove it.